modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/individual_controller.rb

```diff
@@ -32,6 +32,8 @@ def validate
         def validate_individual_poa_code!(poa_code)
           return if ::Veteran::Service::Representative.where('? = ANY(poa_codes)', poa_code).any?
 
+          return if %w[067 072].include? poa_code
```

Comment:
There should be more than two poa codes that are acceptable for 2122a submissions unless I missed this is for a pilot test or something.

I would've expected the check to be something like;
```ruby
return if ::Veteran::Service::Organization.find_by(poa: poa_code).blank?
```

Essentially we're ensuring that the poa code sent in does not associate with any organization